webui/react/src/components/Link.tsx

```diff
@@ -1,43 +1,46 @@
 import React, { MouseEvent, MouseEventHandler, PropsWithChildren, useCallback } from 'react';
 
-import { handlePath, windowOpenFeatures } from 'utils/routes';
+import { handlePath, serverAddress, windowOpenFeatures } from 'utils/routes';
 
 import css from './Link.module.scss';
 
-interface Props {
+export interface LinkProps {
```

Comment:
:thinking: is this named this way for easier import? I've run into this before. do we want to name it here or use 

import {Props as LinkProps} from 'Links".  either works we should just stay consistent.

Reply:
nice, `Props as LinkProps` is so much better!